### PR TITLE
Add option for model normals on ply export

### PR DESCRIPTION
**Pull request by CameronFraser**
This PR makes the arguments related to normals for point cloud export consistent with the arguments related to normals for Poisson export. Doing this adds support for the use case where a user may want to export just a point cloud with predicted normals instead of o3d estimated normals to use in their own Poisson meshing pipeline.

Additionally, the viewer is updated to reflect the changes.

### CODE REVIEW

File: nerfstudio/scripts/exporter.py [hunk 1]
```diff
@@ -59,6 +59,30 @@ class Exporter:
     """Path to the output directory."""
 
 
+def validate_pipeline(normal_method, normal_output_name, pipeline: Pipeline) -> None:
```
Comment (tancik) [inline nerfstudio/scripts/exporter.py:62]: Add types and documentation for the args.

**Review by tancik — COMMENTED** [review]
Thanks for adding. Sorry for the delayed review.

File: nerfstudio/scripts/exporter.py (hunk 1 above)
Comment (CameronFraser) [reply nerfstudio/scripts/exporter.py]: np, I just updated that file

**Comment by ethanweber** [comment]
LGTM. Can you merge `main` now and then we can merge into `main`?

**Comment by CameronFraser** [comment]
@ethanweber updated branch with main

**Review by ethanweber — APPROVED** [review]
Great thanks! LGTM, and I tried out the viewer changes too.

**Comment by CameronFraser** [comment]
worth noting that in my case when using nerfstudio with aerial imagery the nerfacto predicted normals and the resulting mesh were significantly better looking than using open3d

**Pull request merged**